it/src/main/java/org/corfudb/universe/node/server/CorfuServerParams.java

```diff
@@ -14,52 +14,68 @@
 import org.corfudb.universe.node.server.CorfuServer.Persistence;
 import org.slf4j.event.Level;
 
+import java.nio.file.Path;
+import java.nio.file.Paths;
 import java.time.Duration;
+import java.util.Optional;
 import java.util.Set;
 
 @Builder(builderMethodName = "serverParamsBuilder")
 @AllArgsConstructor
 @EqualsAndHashCode(exclude = {"logLevel", "stopTimeout"})
 @ToString
+@Getter
 public class CorfuServerParams implements NodeParams {
+    public static final String DOCKER_IMAGE_NAME = "corfudb-universe/corfu-server";
+
     @NonNull
     private final String streamLogDir = "db";
 
     @Default
-    @Getter
     private final int port = ServerUtil.getRandomOpenPort();
 
     @Default
     @NonNull
-    @Getter
     private final Mode mode = Mode.CLUSTER;
 
     @Default
     @NonNull
-    @Getter
     private final Persistence persistence = Persistence.DISK;
 
     @Default
     @NonNull
-    @Getter
-    private final Level logLevel = Level.DEBUG;
+    private final Level logLevel = Level.INFO;
 
     @NonNull
-    @Getter
     private final NodeType nodeType = NodeType.CORFU_SERVER;
 
     /**
      * A name of the Corfu cluster
      */
-    @Getter
     @NonNull
     private final String clusterName;
 
-    @Getter
     @Default
     @NonNull
     private final Duration stopTimeout = Duration.ofSeconds(1);
 
+    @Default
+    private final Optional<ContainerResources> containerResources = Optional.empty();
+
+    /**
+     * Corfu server version, for instance: 0.3.0-SNAPSHOT
+     */
+    @NonNull
+    private final String serverVersion;
+
+    @NonNull
+    @Default
+    private final Path serverJarDirectory = Paths.get("target");
```

Comment:
Can you make this configurable? That would be pretty useful.

Reply:
it is already configurable
```
wf.setupDocker(fixture -> {
    fixture.getServer(). serverJarDirectory("my_custom_jar_directory");
});
```

Reply:
Actually, everything that you can see in any `Prop` classes can be configured in your test